### PR TITLE
Add function to display comment navigation

### PR DESCRIPTION
**Pull request by IanDelMar**
## Description

Adds the function `understrap_comment_navigation()` to display the comment navigation.

## Motivation and Context

Avoid code duplication.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have checked that there aren't other open Pull Requests for the same update/change.
- [x] My code follows the code style of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] `composer cs:check` has passed locally.
- [x] `composer lint:php` has passed locally.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: inc/template-tags.php [hunk 1]
```diff
@@ -157,3 +157,36 @@ function understrap_body_attributes() {
...
 }
+
+if ( ! function_exists( 'understrap_comment_navigation' ) ) {
+	/**
+	 * Displays the comment navigation.
+	 *
+	 * @param string $nav_id The ID of the comment navigation.
+	 */
+	function understrap_comment_navigation( $nav_id ) {
+		if ( get_comment_pages_count() <= 1 || false === get_option( 'page_comments' ) ) {
```
Comment (IanDelMar) [inline inc/template-tags.php:168]: Can we drop `|| false === get_option( 'page_comments' )`? AFAIK, `get_comment_pages_count() <= 1` will be evaluated first. And `get_comment_pages_count()` includes the following code
```php
    if ( ! get_option( 'page_comments' ) ) {
        return 1;
    }
```
Code reference: https://core.trac.wordpress.org/browser/tags/5.8/src/wp-includes/comment.php#L1034

So, if `get_option( 'page_comments' ) = false` `get_comment_pages_count()` will return 1 and the whole if statement evaluates to true. No?

Comment (danemorgan) [reply inc/template-tags.php:168]: Yes. It looks to me like that condition is already baked in.

Comment (danemorgan) [reply inc/template-tags.php:168]: If you want to push that change, I'll merge this into develop and try it out.

**Review by danemorgan — CHANGES_REQUESTED** [review]

File: inc/template-tags.php (hunk 1 above)
Comment (danemorgan) [reply inc/template-tags.php:168]: if you want to push that change, I'll merge it into develop.

**Review by danemorgan — CHANGES_REQUESTED** [review]
Line 168 from

`if ( get_comment_pages_count() <= 1 || false === get_option( 'page_comments' ) ) {`

to

`if ( get_comment_pages_count() <= 1 ) {`

**Review by danemorgan — APPROVED** [review]

**Pull request merged**